src/main/java/org/spongepowered/api/event/entity/EntityDeathEvent.java

```diff
@@ -27,11 +27,42 @@
 
 import org.spongepowered.api.entity.Entity;
 import org.spongepowered.api.event.cause.CauseTracked;
+import org.spongepowered.api.item.inventory.ItemStack;
 import org.spongepowered.api.util.event.Cancellable;
 
+import java.util.List;
+
 /**
  * Called when an {@link Entity} is killed or removed due to unload.
  */
 public interface EntityDeathEvent extends EntityEvent, CauseTracked, Cancellable {
+
+    /**
+     * Gets a copy of the {@link ItemStack}s this entity will drop on death.
+     *
+     * @return A copy of the list of itemstacks the entity will drop
+     */
+    List<ItemStack> getDrops();
+
+    /**
+     * Sets the list of {@link ItemStack}s the entity will drop on death.
+     *
+     * @param drops The list of drops the entity will drop on death
+     */
+    void setDrops(List<ItemStack> drops);
```

Comment:
I think the player will lose everything on death, and this is what it drops. (Correct me if I am wrong)